src/Impostor.Server/Net/Inner/Objects/ShipStatus/InnerPolusShipStatus.cs

```diff
@@ -36,13 +36,15 @@ public override Vector2 GetSpawnLocation(InnerPlayerControl player, int numPlaye
             }
 
             Vector2 position;
-            if (player.PlayerId < 5)
+            int halfPlayers = numPlayers / 2; // floored intentionally
+            int spawnId = player.PlayerId % 15;
```

Comment:
Is this not going to have the same issues voting had with NetId? Where if you disconnect and connect to the lobby your PlayerId/netId and order do not match anymore?